whois-query/src/main/java/net/ripe/db/whois/query/acl/AccountingIdentifier.java

```diff
@@ -5,18 +5,20 @@
 public class AccountingIdentifier {
 
     final private InetAddress remoteAddress;
-    final private String userName;
+    final private UserInfo ssoUser;
 
-    public AccountingIdentifier(InetAddress remoteAddress, String userName) {
+    public AccountingIdentifier(InetAddress remoteAddress, UserInfo ssoUser) {
         this.remoteAddress = remoteAddress;
-        this.userName = userName;
+        this.ssoUser = ssoUser;
     }
 
-    public String getUserName() {
-        return userName;
+    public UserInfo getSsoUser() {
+        return ssoUser;
     }
 
     public InetAddress getRemoteAddress() {
         return remoteAddress;
     }
+
+    public record UserInfo(String userName, String uuid) {}
```

Comment:
Is it possible to send the list of maintainers for a uuid as parameter to requiresAcl? AccountingIdentifier class  should have only those fields for which we do accounting. This function  requiresAcl is being called from two places only from a queryHandler and ElasticSearchCallback. This also helps to keep the accessListManager class granular too i.e it just do counting for personal objects and related operations  rather than hitting the database and fetching the maintainers too.